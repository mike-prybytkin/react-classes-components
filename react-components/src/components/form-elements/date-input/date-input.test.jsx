import React from 'react';
import { render, screen } from '@testing-library/react';
import mockText from '../../../mocks/text';
import DateInput from './date-input';

const setUp = () => {
  return render(
    <DateInput
      labelType={mockText.labelUserBirthday}
      defaultValue="2000-01-01"
      inputDateRef={undefined}
    />
  );
};

describe('DateInput component', () => {
  // it('should correctly render date input', () => {
  //   setUp();
  //   expect(screen.findByText(new RegExp(mockText.labelUserBirthday, 'i'))).toBeInTheDocument();
  // });

  it('should create snapshot', () => {
    const { asFragment } = setUp();
    expect(asFragment()).toMatchSnapshot();
  });
});
