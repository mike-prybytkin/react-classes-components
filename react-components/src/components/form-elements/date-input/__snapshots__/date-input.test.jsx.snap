// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DateInput component should create snapshot 1`] = `
<DocumentFragment>
  <label
    class="user-birthday__label"
  >
    Birthday:
    <input
      class="user-birthday__input"
      type="date"
      value="2000-01-01"
    />
  </label>
</DocumentFragment>
`;
